openapi/openapi.yaml

```diff
@@ -620,8 +620,16 @@ components:
           description: The URL for this tool in this registry.
         id:
           type: string
-          example: 123456
-          description: A unique identifier of the tool, scoped to this registry.
+          example: dockstore:123456
+          description: A unique identifier of the tool, scoped to this registry with a
+            CURIE-like prefix. See
+            https://github.com/ga4gh/tool-registry-service-schemas/blob/develop/registry.json
+            for the current list of prefixes and the registry that they
+            correspond to. TRS IDs are strings made up of uppercase and
+            lowercase letters, decimal digits, hypen, period, percent,
+            underscore and tilde [A-Za-z0-9.-%_~] to simplify parsing CURIE-like
+            prefixes. Special characters like slashes can be encoded to work
+            with these restrictions.
```

Comment:
The more I think about it, I'm not sure, but I would lean towards `within` instead of `with`.